distribution/packages/deb/build.gradle

```diff
@@ -1,61 +0,0 @@
-import org.elasticsearch.gradle.LoggedExec
-
-/*
```

Comment:
I think we should keep the license header here too.

And a comment that the actual build is configured by the parent.